async-query-core/src/test/java/org/opensearch/sql/spark/asyncquery/AsyncQueryCoreIntegTest.java

```diff
@@ -185,7 +185,7 @@ public void setUp() {
   public void createDropIndexQuery() {
     givenSparkExecutionEngineConfigIsSupplied();
     givenValidDataSourceMetadataExist();
-    when(queryIdProvider.getQueryId(any())).thenReturn(QUERY_ID);
+    when(queryIdProvider.getQueryId(any(), eq(asyncQueryRequestContext))).thenReturn(QUERY_ID);
```

Comment:
nit: any reason why using both any() and eq()? Does any() suffice, or do we also want eq(dispatchQueryRequest)?
asking because I think it might be a bit confusing in future when reading this piece. Not really a concern tho

Reply:
The first parameter is verified using ArgmentCaptor later since we want to verify the attributes inside the object.